### PR TITLE
[PRTL-3044] adding tabs to scrna-seq

### PR DESCRIPTION
**Pull request by samrichca**
## Ticket Number

PRTL-3044

## Environment to be used in testing

- [x] `prod`
- [ ] `dev-oicr`
- [ ] `qa` (which version?)

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/ExploreCasesTable/ExploreCasesTable.model.js [hunk 1]
```diff
@@ -288,25 +317,32 @@ const casesTableModel = [
...
                       ]),
                     }}
-                  >
-                    <MicroscopeIcon style={{ maxWidth: '20px' }} /> ({slideCount})
+                    >
+                    <MicroscopeIcon style={{ maxWidth: '20px' }} />
+                    {' '}
+(
+                    {slideCount}
+)
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ExploreCasesTable/ExploreCasesTable.model.js]:
```suggestion

                    {` (${slideCount})`}
```

File: src/packages/@ncigdc/modern_components/SCRNASeq/SCRNASeq.js [hunk 2]
```diff
@@ -1,89 +1,122 @@
...
-    const { data = [], dataType = '' } = this.state;
-
-    return (
-      <Column style={{ marginBottom: '1rem' }}>
-        <Row
-          style={{
-            margin: '20px 0',
-            padding: '2rem 3rem',
+const SCRNASeq = ({
+  activeTab, data, dataType, setActiveTab, setDataType,
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/SCRNASeq.js]:
```suggestion
  activeTab,
  data,
  dataType,
  setActiveTab,
  setDataType,
```

File: src/packages/@ncigdc/modern_components/SCRNASeq/plots/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import MedianGenesPlot from './MedianGenesPlot';
+import SCRNASeqPlot from './SCRNASeqPlot';
+import SequencingSaturationPlot from './SequencingSaturationPlot';
+
+export { MedianGenesPlot, SCRNASeqPlot, SequencingSaturationPlot };
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/plots/index.js]:
```suggestion
export { default as MedianGenesPlot } from './MedianGenesPlot';
export { default as SCRNASeqPlot } from './SCRNASeqPlot';
export { default as SequencingSaturationPlot } from './SequencingSaturationPlot';
```

File: src/packages/@ncigdc/modern_components/SCRNASeq/tables/ClusterTable.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+import EntityPageHorizontalTable from '@ncigdc/components/EntityPageHorizontalTable';
+import { random } from 'lodash';
+
+const ClusterTable = () => {
+  const stubClusterArray = [...Array(11)].map((arr, i) => ({
+    key: `cluster-${i + 1}`,
+    subheadings: ['L2FC', 'p-value'],
+    title: `Cluster ${i + 1}`,
+  }));
+  const stubHeadings = [
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/tables/ClusterTable.js]:
```suggestion

  const stubHeadings = [
```

File: src/packages/@ncigdc/modern_components/SCRNASeq/tables/index.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import ClusterTable from './ClusterTable';
+
+export { ClusterTable };
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/tables/index.js]:
```suggestion
export { default as ClusterTable } from './ClusterTable';
```

File: src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/DownloadButton.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+export default class DownloadButton extends Component {
+  handleClick = e => {
+    const { onToolbarClick } = this.props;
+    onToolbarClick(e);
+  }
+
+  render() {
+    const {
+      faClass, label, name,
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/DownloadButton.js]:
```suggestion
      faClass,
      label,
      name,
```

File: src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/index.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import DownloadButton from './DownloadButton';
+import ToolbarButton from './ToolbarButton';
+
+export {
+  DownloadButton, ToolbarButton,
+};
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/index.js]:
```suggestion
export { default as DownloadButton } from './DownloadButton';
export { default as ToolbarButton } from './ToolbarButton';
```

Comment (samrichca) [reply src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/index.js]: does this change how I import these modules?

Comment (caravinci) [reply src/packages/@ncigdc/modern_components/SCRNASeq/toolbar/index.js]: Not in the slightest. This syntax is just a more concise version of what you wrote, without bridging the referred file without bringing their code into this context.

**Pull request merged**